2019/2019-03-12.md

```diff
@@ -70,7 +70,6 @@ Start: 2:00 pm
         > [<u>https://trello.com/c/nSPx2UJX/235-light-microscopy-australia-09-03-2019</u>](https://trello.com/c/nSPx2UJX/235-light-microscopy-australia-09-03-2019)
 
     -   Fun in Singapore:
-        > <img src="../images/2019-03-12-image1.png" style="width:2.95313in;height:1.65849in" />
```

Comment:
These images _are_ present, but I'm not sure why they aren't being found:

```
commit 9806d5586726db3bf29d4496e83c4bb367cecf6c
Author: jmoore <josh@glencoesoftware.com>
Date:   Tue Sep 29 21:40:55 2020 +0200

    More image fixes
```

Reply:
Based on https://github.com/ome/www.openmicroscopy.org/actions/runs/3219662885/jobs/5265329745, the issue with this image is the absence of an `alt` attribute rather than the fact the source is missing.

Reply:
Yes, @sbesson is right, @joshmoore I am removing them because I do not have an easy solution about adding an ``alt`` attribute. It seems that this needs javascript abilities, or somesuch ?

Reply:
thanks @joshmoore fixed in https://github.com/ome/meeting-minutes/pull/5/commits/e08d85af479a7b8d7aab7c03831add0d3ff0bc85